_api-reference/validate.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: default
+title: Validate Query
+nav_order: 87
+---
+
+# Validate Query
+
+You can use the Validate Query API to validate a query without running it. The query can be sent as a path parameter or in the request body.
```

Comment:
```suggestion
You can use the Validate Query API to validate a query without running it. The query can be sent as a path parameter or inside the request body.
```

Reply:
Disagree. "in" is sufficient/accurate here. Also, how about "or included in the request body."

Reply:
included works for me.